homeassistant/components/met/__init__.py

```diff
@@ -22,7 +22,7 @@
 
 from .const import CONF_TRACK_HOME, DOMAIN
 
-URL = "https://api.met.no/weatherapi/locationforecast/2.0/classic"
+URL = "https://api.met.no/weatherapi/locationforecast/2.0/complete"
```

Comment:
```suggestion
```
This is the default in the lib, it can be removed.
The URL on L145 should also be removed.

Reply:
Updated